addons/weaponselect/stringtable.xml

```diff
@@ -288,6 +288,12 @@
             <Chinese>選擇非破片手榴彈</Chinese>
             <Turkish>El Bombası Dışındakileri Seç</Turkish>
         </Key>
+        <Key ID="STR_ACE_WeaponSelect_SelectGrenadeFlash">
+            <English>Select Flashbang</English>
+        </Key>
+        <Key ID="STR_ACE_WeaponSelect_SelectGrenadeIncendiary">
+            <English>Select Incendiary Grenade</English>
+        </Key>
```

Comment:
Add missing strings and russian translation. Also I think propper naming is Stun Grenade. Maybe key id should be changed too.

```suggestion
        <Key ID="STR_ACE_WeaponSelect_SelectGrenadeFlash">
            <English>Select Stun Grenade</English>
            <Russian>Выбрать светошумовую гранату</Russian>
        </Key>
        <Key ID="STR_ACE_WeaponSelect_SelectGrenadeIncendiary">
            <English>Select Incendiary Grenade</English>
            <Russian>Выбрать зажигательную гранату</Russian>
        </Key>
        <Key ID="STR_ACE_WeaponSelect_SelectGrenadeSmoke">
            <English>Select Smoke Grenade</English>
            <Russian>Выбрать дымовую гранату</Russian>
        </Key>
        <Key ID="STR_ACE_WeaponSelect_SelectGrenadeFlare">
            <English>Select Flare Grenade</English>
            <Russian>Выбрать осветительную гранату</Russian>
        </Key>
```